verification/initializeContext_test.go

```diff
@@ -11,17 +11,25 @@ import (
 // This file is used to test the initialize context command by using a simulator
 
 func TestInitializeContext(t *testing.T) {
-	simulators := []TestDPEInstance{
-		// No extra options.
-		&DpeSimulator{exe_path: *sim_exe},
-		// Supports simulation.
-		&DpeSimulator{exe_path: *sim_exe, supports: Support{Simulation: true}},
+	var instances []TestDPEInstance
+	if *isEmulator {
+		//Added dummy support for emulator. Once the emulator is implemented, will add the actual enabled feature
+		instances = []TestDPEInstance{
+			&DpeInstance{exe_path: *socket_exe, supports: Support{AutoInit: true}},
+		}
+	} else {
+		instances = []TestDPEInstance{
+			// No extra options.
+			&DpeInstance{exe_path: *socket_exe},
+			// Supports simulation.
+			&DpeInstance{exe_path: *socket_exe, supports: Support{Simulation: true}},
```

Comment:
I think we may want to change how we do this. Instead, I'm thinking:

* Each test may require specific `support` features to be enabled
* If the test target matches the requires support, the test will run
* If the target doesn't match, we skip the test

This will mean we have more test cases, but it will be easier to run the tests against targets that have different support flags set.

We can have a GetTestTarget function that does something like this:

```go
fun GetTestTarget(support_needed Support) (error, TestDPEInstance) {
  if *isEmulator {
    // Check if all the support==True are supported in the emulator. If not, return error. If yes, return a DpeEmulator instance
  } else {
    // Make a DpeSimulator with the requested support
  }
}
```

Reply:
Here, assumed like a fixed set of support features(support_needed) has been enabled in DpeEmulator struct and checked the Boolean value for the support needed when returning DPE instance for the emulator. Please check the GetEmulatorTarget code and provide any comments you may have.